client/js/src/main/scala/zio/zmx/client/frontend/views/DiagramView.scala

```diff
@@ -45,40 +107,16 @@ object DiagramView {
             }),
             cls := "bg-gray-900 text-gray-50 rounded my-3 p-3",
             span(
-              cls := "text-2xl font-bold my-2",
+              cls := "w-full flex items-center justify-center text-2xl font-bold my-2",
```

Comment:
Just minor - I think having the title on the left hand side is better. I would move the up/down/close button to the top right corner of the diagram view (like the window controls in some UIs)